### ISSUE TITLE
Don't localize command names on the command palette

### ISSUE THREAD

**Issue by espresso3389** [microsoft/terminal#7039]
# Environment

```none
Windows build number: 10.0.20175.0 (Japanese)
Windows Terminal version: 1.2.2022.0
```
# Steps to reproduce/Actual behavior

Opening command palette and type some command in English.
For example, if I want to invoke certain `scroll` commands, I want to search the commands with typing `scro...` but because Windows terminal localizes these command strings, I cannot find the commands.

![cpal](https://user-images.githubusercontent.com/1311400/88288452-bf4e7080-cd2e-11ea-8f88-e100e5ba28f7.gif)

# Expected behavior

Please consider some way to allow us (especially CJK users) to search the commands in plain English. Typing CJK words are not so easy and if the commands are localized, it makes the command palette almost unusable.

VSCode have had similar issues before and they fixed the issue already:

- https://github.com/microsoft/vscode/issues/4679
- https://github.com/microsoft/vscode/issues/4691

**Comment by zadjii-msft** [comment]
Thanks for the feedback! I didn't know that VSCode's command palette did that, so this is helpful feedback. I'm gonna stick this into 2.0 for now.

For those who want to skip to the resolution of the above thread: https://github.com/microsoft/vscode/issues/4679#issuecomment-213170197

**Comment by espresso3389** [comment]
FYI, I also attach VSCode Japanese locale screenshot that shows how they deal with the command palette localization problem:

![image](https://user-images.githubusercontent.com/1311400/88305654-b5d00300-cd44-11ea-80ec-6e2fa780b150.png)
